tokens/mpeth.json

```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x48afbbd342f64ef8a9ab1c143719b63c2ad81710",
+    "near_address": "48afbbd342f64ef8a9ab1c143719b63c2ad81710.factory.bridge.near",
+    "aurora_address": "0xb64079dc31e0b28919e506d00f5cc50a74e7cabb",
+    "name": "MetaPoolETH",
+    "symbol": "MPETH",
+    "decimals": 18,
+    "icon": "data:image/svg+xml,%3Csvg%20width%3D%22204%22%20height%3D%22204%22%20viewBox%3D%220%200%20204%20204%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Crect%20x%3D%220.25%22%20y%3D%220.235352%22%20width%3D%22203.501%22%20height%3D%22203.501%22%20rx%3D%22101.751%22%20fill%3D%22%23BAC2E2%22%2F%3E%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M102.001%2039.6226L86.8763%2054.54L102.001%2069.4575L117.126%2054.54L102.001%2039.6226ZM77.8967%2063.2419L63.257%2077.9581L101.794%20116.696L140.331%2077.9581L125.691%2063.242L101.794%2087.2638L77.8967%2063.2419ZM54.6541%2086.8613L39.6377%20101.899L102.001%20164.349L164.364%20101.899L149.348%2086.8614L102.001%20134.274L54.6541%2086.8613Z%22%20fill%3D%22%230C2246%22%2F%3E%3C%2Fsvg%3E",
```

Comment:
Could you please use base64 encoding.
https://codebeautify.org/svg-to-base64-converter

Reply:
Sure. Let me know if it is fine now. Thanks!